traitquestdev/css/style.css

```diff
@@ -26,11 +26,12 @@ select{}
 textarea{resize:none;}
 #logo-image{max-height:60px;}
 .clear:after {clear: both;content: "";display: table;}
-.inputForm{padding-top: 8px; padding-bottom: 8px;}
+.inputForm{width: 200px; padding:8px; display:block; margin: 0 auto;}
 .columnInput{margin-bottom: 5px;}
 .button{padding-top: 8px; padding-bottom: 8px; border: none;color:white; background-color: #2ECC71; cursor: pointer;transition: all 0.5s ease;-moz-transition: all 0.5s ease;-webkit-transition: all 0.5s ease;-o-transition: all 0.5s ease;-ms-transition: all 0.5s ease;}
```

Comment:
please use the color in the sketch that i put in dropbox, i think the color code should be different.